index.html

```diff
@@ -2,16 +2,16 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <title>Fork Me! FCC: Test Suite Template</title>
-    <link rel="stylesheet" href="css/style.css" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fork Me FCC: Test Suite Template</title>
   </head>
-
```

Comment:
- It is advisable to remove all blank lines without any code, use extensions in your code editor to help you with formatting lines of code, for easy work efficiency. 👍

Reply:
Nice job including an explanation, a "why", for each requested change. However, It would be better to phrase the comments politely, as suggestions instead of orders. For example:

* This is a polite suggestion: *"I think it would be better to use extensions in your code editor to help you with formatting lines of code, for easy work efficiency. 👍"*
* This reads like an order: *"use extensions in your code editor to help you with formatting lines of code, for easy work efficiency. 👍"*